app/services/random_phrase.rb

```diff
@@ -9,7 +9,7 @@ def initialize(num_words:, word_length: WORD_LENGTH)
   end
 
   def to_s
-    @words.join(' ')
+    @words.join('-')
```

Comment:
this could be unchanged if you let the `PersonalKeyGenerator` make the `tr` in `create`.